### PR TITLE
Shorten Commands

### PR DESCRIPTION
**Pull request by RobiNino**
Sample of the new help command under the main namespace:
```
NAME:
   jfrog - See https://github.com/jfrog/jfrog-cli for usage instructions.

USAGE:
   jfrog [global options] command [command options] [arguments...]

VERSION:
   2.5.1

COMMANDS:
     help, h  Shows a list of commands or help for one command

   Audit/Scan Commands:
     audit-go, ago     Execute an audit Go command, using the configured Xray details.
     audit-gradle, ag  Execute an audit Gradle command, using the configured Xray details.
     audit-mvn, am     Execute an audit Maven command, using the configured Xray details.
     audit-npm, an     Execute an audit Npm command, using the configured Xray details.
     audit-pip, ap     Execute an audit Pip command, using the configured Xray details.
     scan, s           Scan files located on the local file-system with Xray.

   BuildTools Commands:
     dotnet                  Run .NET Core CLI
     dotnet-config, dotnetc  Generate dotnet configuration.
     go, go                  Runs go
     go-config, goc          Generate go build configuration.
     go-publish, gp          Publish go package and/or its dependencies to Artifactory
     gradle                  Run Gradle build.
     gradle-config, gradlec  Generate gradle build configuration.
     mvn                     Run Maven build.
     mvn-config, mvnc        Generate maven build configuration.
     npm                     Run npm command.
     npm-config, npmc        Generate npm configuration.
     nuget                   Run NuGet.
     nuget-config, nugetc    Generate nuget configuration.
     pip                     Run pip install.
     pip-config, pipc        Generate pip build configuration.
     yarn                    Run Yarn commands.
     yarn-config, yarnc      Generate Yarn configuration.

   CI Setup Commands:
     ci-setup  Set up a CI pipeline with the JFrog Platform

   Plugins:
     rt-fs  Run file system commands in Artifactory.

   Subcommands:
     rt          Artifactory commands
     mc          Mission Control commands
     xr          Xray commands
     ds          Distribution commands
     completion  Generate autocomplete scripts
     plugin      Plugins handling commands
     config, c   Config commands

GLOBAL OPTIONS:
   --help, -h     show help
   --version, -v  print the version
```

Audit/Scan Commands specify the commands that were moved from the 'xr' namespace.
BuildTools Commands specify the commands that were moved from the 'rt' namespace.
Of all the above, all commands remained the same other than npm & pip that were modified to a native syntax.

Older syntax is still accepted but is now deprecated. A deprecated warning will be displayed when using the old syntax, which can be suppressed by setting the `JFROG_CLI_AVOID_DEPRECATION_WARNINGS` env var to true.

Depends on [jfrog/jfrog-cli-core#251](https://github.com/jfrog/jfrog-cli-core/pull/251)

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
I haven't reviewing everything yet. Will do so soon, but here's my initial suggestions for the commands help titles - 

Audit/Scan Commands: --> Audit & scan

BuildTools Commands: --> Build tools

Let's merge "CI Setup Commands:" and "Subcommands:" under "Other".

Let's put "Plugins:" last.

File: docs/buildtools/gradle/help.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package gradle
+
+const Description = "Run Gradle build."
+
+var Usage = []string{`jfrog gradle <tasks and options> [command options]`}
+
+const Arguments string = `	tasks and options
+		Tasks and options to run with gradle command. For example, -b path/to/build.gradle.`
+
+const EnvVar string = `	JFROG_CLI_EXTRACTORS_REMOTE
```
Comment (eyalbe4) [inline docs/buildtools/gradle/help.go:10]: I suggest consulting with @sverdlov93 , to see if following the new "options" command he's working on, the env vars should still be here. The same goes for the other files which include env vars.

**Comment by RobiNino** [comment]
We can't control the order, they are sorted alphabetically.
But after these adjustment "Plugins" happen to be last :)

File: docs/auditscan/scan/help.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package scan
+
+const Description = "Scan files located on the local file-system with Xray."
+
+var Usage = []string{"jfrog scan [command options] <source pattern> ",
```
Comment (eyalbe4) [inline docs/auditscan/scan/help.go:5]: I think we should have a const in the code, that will be initialized to either "jfrog" or "jf" according to the executable name. We can then use this constant in all help pages, such as the above.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: docs/auditscan/auditgo/help.go [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package auditgo
```
[the remark this answers is not in the document]
Comment (RobiNino) [reply docs/auditscan/auditgo/help.go:1]: The purpose for this structure is to allow all help variable to be named the same (Description, Usage, Arguments).
Putting all help.go files under the same package will require renaming all these variable as golang complains about "redeclared in this package"

**Pull request merged**